docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -0,0 +1,145 @@
+module Scripts.Holding where
+
+import DA.Map (empty, fromList)
+import DA.Set (singleton)
+import Daml.Script
+
+-- INTERFACE DEPENDENCIES --
+import Daml.Finance.Interface.Account.Factory qualified as Account (F)
+import Daml.Finance.Interface.Holding.Base qualified as Holding (I)
+import Daml.Finance.Interface.Holding.Factory qualified as Holding (F)
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (I)
+import Daml.Finance.Interface.Types.Common.Types (AccountKey, Id(..), InstrumentKey(..))
+
+-- IMPLEMENTATION DEPENDENCIES --
+import Daml.Finance.Account.Account qualified as Account (Factory(..))
+import Daml.Finance.Holding.Fungible qualified as Fungible (Factory(..), Fungible(..))
+import Daml.Finance.Instrument.Token.Instrument (Instrument(..))
+
+import Workflow.CreateAccount qualified as CreateAccount
+import Workflow.CreditAccount qualified as CreditAccount
+
+-- | Helper container used to transfer state from one script to another.
+data HoldingState = HoldingState
+  with
+    alice : Party
+    bank : Party
+    bob : Party
+    public : Party
+    aliceAccount : AccountKey
+    bobAccount : AccountKey
+    cashInstrument : InstrumentKey
+    holdingFactoryCid : ContractId Holding.F
+    aliceCashHoldingCid : ContractId Holding.I
+  deriving (Eq, Show)
+
+-- | Test script that
+-- 1. creates an account for Alice and Bob at the Bank
+-- 2. issues a cash instrument
+-- 3. credits a cash holding to Alice in her bank account
+-- 4. transfers the holding from Alice to Bob
+runHolding : Script HoldingState
+runHolding = do
+
+  -- Allocate parties
+  [alice, bank, bob, public] <- mapA createParty $ ["Alice", "Bank", "Bob", "Public"]
```

Comment:
Do you need the `$` function here ? I'm just conscience that this script is intended for new learners.

Reply:
No, this was redundant. I have removed it here and in another file as well now.